sites/all/modules/custom/practice/practice.module

```diff
@@ -0,0 +1,206 @@
+<?php
+
+/**
+ * @file
+ * This is drupal 7 practice module.
+ */
+
+/**
+ * Represents max amount of latest nodes.
+ */
+define('PRACTICE_MAX_NODE_AMOUNT', 5);
+
+/**
+ * Represents max length of node title.
+ */
+define('PRACTICE_MAX_TITLE_LENGTH', 50);
+
+/**
+ * Implements hook_menu().
+ */
+function practice_menu() {
+  $items['admin/node-titles'] = array(
+    'title' => 'Latest node\'s titles',
+    'description' => 'Form with 5 latest node\'s titles',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('practice_latest_nodes_form'),
+    'access arguments' => array('access node tittle form'),
+    'type' => MENU_NORMAL_ITEM,
+  );
+  return $items;
+}
+
+/**
+ * Implements hook_permission().
+ */
+function practice_permission() {
+  return array(
+    'access node title form' => array(
+      'title' => t('Access node tittle form'),
+      'description' => t('Access form with titles of latest 5 nodes.'),
+    ),
+  );
+}
+
```

Comment:
remove blank line (1 blank line should be used to separate PHP language structures)

Reply:
removed